src/main/scala/info/bethard/timenorm/formal/Types.scala

```diff
@@ -1,13 +1,19 @@
 package info.bethard.timenorm.formal
 
-import java.time.temporal.{Temporal, TemporalAmount, TemporalField, TemporalUnit}
+import java.time.temporal._
 import java.time.LocalDateTime
 import java.util
 
+import scala.collection.JavaConverters._
+
 trait TimeExpression
 
 trait Number extends TimeExpression
-case class IntNumber(n: Int) extends Number
+case class IntNumber(n: Int) extends Number {
+
+  def getValue( ): Int = {  return n }
```

Comment:
For your current implementation, you don't need this method, so leave it out for now. We should discuss if there's a common method that all Number subclasses should support.